pkg/blackduck/ctl_blackduck.go

```diff
@@ -370,6 +374,18 @@ func (ctl *Ctl) SetFlag(f *pflag.Flag) {
 			ctl.Spec.PostgresPassword = crddefaults.Base64Encode([]byte(ctl.PostgresPassword))
 		case "user-password":
 			ctl.Spec.UserPassword = crddefaults.Base64Encode([]byte(ctl.UserPassword))
+		case "enable-binary-analysis":
+			if strings.ToUpper(ctl.EnableBinaryAnalysis) == "TRUE" {
```

Comment:
As a BoolVar this check can be GetFlagBool

Reply:
@rrati Since I don't have the flagset in this function I cant use the GetFlagBool(). I can do f.Value.String() and then cast it to a bool, but this will also impair how we test this functionality. Otherwise, checking the value in ctl.EnableBinaryAnalysis works and keeps our test cases working.